source/LoginTarget.js

```diff
@@ -140,12 +140,12 @@ export default class LoginTarget {
      *      loginTarget.enterDetails("myUsername", "myPassword");
      */
     enterDetails(username, password) {
-        this.usernameFields.forEach(field => {
+        this.usernameFields.slice(0, 1).forEach(field => {
```

Comment:
totally unrelated, but how nice would an `if (let field = usernameFields[0]) { ... }` syntax be 😍

Reply:
Haha, agreed! That'd be clean..